README.md

```diff
@@ -0,0 +1,31 @@
+# Found Your Stuff Core
+
+This is the repository for the backend API for the Found Your Stuff (FYS) item tracking system. 
```

Comment:
This is the repository for the backend API for the Found Your Stuff (FYS) item tracking system. -> Backend API of the Found Your Stuff (FYS) project's item tracking system.